metadata.json

```diff
@@ -27,14 +27,6 @@
         "16.10",
         "17.04"
       ]
-    },
-    {
-      "operatingsystem": "Linux Mint",
```

Comment:
Right now this stops rspec-puppet-facts from working as LinuxMint facts are missing:

https://github.com/camptocamp/facterdb/issues/41